db/models.py

```diff
@@ -49,4 +57,52 @@ class MovieSession(models.Model):
     )
 
     def __str__(self) -> str:
-        return f"{self.movie.title} {str(self.show_time)}"
+        return f"{self.movie.title} {self.show_time}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to=get_user_model(),
+                             on_delete=models.CASCADE,
+                             related_name="orders")
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+    class Meta:
+        ordering = ("-created_at",)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(MovieSession, on_delete=models.CASCADE)
+    order = models.ForeignKey(Order, on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def __str__(self) -> str:
+        return (f"{self.movie_session.movie.title} "
+                f"{self.movie_session.show_time} "
+                f"(row: {self.row}, seat: {self.seat})")
+
+    def clean(self) -> None:
+        if self.row > self.movie_session.cinema_hall.rows:
+            raise ValidationError({
+                "row": [f"row number must be "
+                        f"in available range: "
+                        f"(1, rows): (1, "
+                        f"{self.movie_session.cinema_hall.rows})"]
+            })
+        if self.seat > self.movie_session.cinema_hall.seats_in_row:
+            raise ValidationError({
+                "seat": [f"seat number must be in available "
+                         f"range: (1, seats_in_row): (1, "
+                         f"{self.movie_session.cinema_hall.seats_in_row})"]})
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
+        super().save(*args, **kwargs)
+
+    class Meta:
+        constraints = [UniqueConstraint(
+            fields=["row", "seat", "movie_session"],
+            name="unique_movie_session")]
```

Comment:
The `class Meta` should be defined after all attributes in the `Ticket` class. According to the checklist, implement `class Meta` only after you defined all attributes.